src/tests/fuzzer/target.cpp

```diff
@@ -981,6 +1108,8 @@ static const std::map<std::string, int (*)(const char*, std::size_t)> g_targets
     { "GetGitBranchFromJsonConfig.", GetGitBranchFromJsonConfig_target },
     { "CheckOrEnsureUsersDontHaveDotFiles.", CheckOrEnsureUsersDontHaveDotFiles_target },
     { "CheckUserAccountsNotFound.", CheckUserAccountsNotFound_target },
+    { "ComplianceEvaluateAudit.", ComplianceEvaluateAudit_target },
+    { "ComplianceEvaluateRemediation.", ComplianceEvaluateRemediation_target },
```

Comment:
>, [](http://example.com/codeflow?start=78&length=1)

bit: ',' not necessary on last list element

Reply:
I would suggest we keep it though - that way we don't need to modify two lines when adding an extra entry to the map (one for the new entry and one adding a comma after the last entry).
So we keep the diffs more readable.